web/ui/react-app/src/pages/LabelsTable.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { FC, useState } from 'react';
+import { RouteComponentProps } from '@reach/router';
+import { Button, Badge, Table } from 'reactstrap';
+
+interface LabelProps {
+  value: Record<string, Record<string, string>>[];
+  name: string;
+}
+
+const printLabels = (labels: Record<string, string>) => {
+  return Object.entries(labels).map(([key, value]) => {
+    return (
+      <div key={key}>
+        <Badge color="primary" className="mr-1">
+          {`${key}="${value}"`}
+        </Badge>
+      </div>
+    );
+  });
+};
+
+export const LabelsTable: FC<RouteComponentProps & LabelProps> = ({ value, name }) => {
+  const [showMore, doToggle] = useState(false);
+
+  const toggleMore = () => {
+    doToggle(!showMore);
+  };
+
+  return (
+    <div className="label-component">
+      <span className="target-head"> {name} </span>
+      <Button size="sm" color="primary" onClick={toggleMore}>
```

Comment:
When a group is expanded, the button touches the table below it without any spacing. But probably another comment that won't matter after adopting the target page header styles :)

Reply:
I think, this was handled by `table-outer-layer`. Anyways, we could add a custom style to the label button as well.

Reply:
I think the proper way is to group the ```span``` and ```button``` in a div and just to apply bootstrap class ```mb-2``` to it